diff --git a/shared/core.js b/shared/core.js
index a7e1277a16e5340372fee1891090dde971d3f229..cc39ff5fb8270357fe4a3070db63e82de98cd666 100644
--- a/shared/core.js
+++ b/shared/core.js
@@ -6,7 +6,12 @@ module.exports = {
     ecmaVersion: 2022,
     ecmaFeatures: { impliedStrict: true, jsx: true },
   },
-  env: { es2022: true, jest: true },
+  env: {
+    // es2022: true,
+    // Workaround for https://github.com/eslint/eslint/issues/15580#issuecomment-1030877816
+    es2021: true,
+    jest: true
+  },
   globals: {
     console: false,
     exports: false,